saleor/graphql/order/mutations/orders.py

```diff
@@ -225,6 +229,17 @@ def get_instance(cls, info, **data):
             )
         return instance
 
+    @classmethod
+    def should_invalidate_prices(cls, instance, cleaned_input, new_instance) -> bool:
+        return any(
+            cleaned_input.get(field) is not None
+            for field in [
+                "shipping_address",
+                "billing_address",
+                "shipping_method",
```

Comment:
Is it correct? I don't see shipping_method in input for this method.

Reply:
Resolved in recent commit.